sdk/eventhub/event-hubs/src/eventHubConsumerClient.ts

```diff
@@ -51,41 +58,64 @@ export class EventHubConsumerClient {
 
   /**
    * @property
-   * The name of the default consumer group in the Event Hubs service.
+   * The default name for the default consumer group in the Event Hubs service when none is
+   * provided during the creation of the consumer group.
    */
   static defaultConsumerGroupName: string = Constants.defaultConsumerGroup;
 
   private _checkpointStore: CheckpointStore;
   private _userChoseCheckpointStore: boolean;
 
+    /**
+   * @property
+   * @readonly
+   * The name of the Event Hub instance for which this client is created.
+   */
+  get eventHubName(): string {
+    return this._eventHubClient.eventHubName;
+  }
+
+  /**
+   * @property
+   * @readonly
+   * The fully qualified namespace of the Event Hub instance for which this client is created.
+   * This is likely to be similar to <yournamespace>.servicebus.windows.net.
+   */
+  get fullyQualifiedNamespace(): string {
+    return this._eventHubClient.fullyQualifiedNamespace;
+  }
+
   /**
    * @constructor
+   * The `EventHubConsumerClient` is the main point of interaction for consuming events from an
+   * Event Hub instance. Use the `options` parmeter to configure retry policy or proxy settings.
    * @param consumerGroup The name of the consumer group from which you want to process events.
-   * @param connectionString - The connection string to use for connecting to the Event Hubs namespace.
+   * @param connectionString - The connection string to use for connecting to the Event Hub instance.
    * It is expected that the shared key properties and the Event Hub path are contained in this connection string.
    * e.g. 'Endpoint=sb://my-servicebus-namespace.servicebus.windows.net/;SharedAccessKeyName=my-SA-name;SharedAccessKey=my-SA-key;EntityPath=my-event-hub-name'.
    * @param options - A set of options to apply when configuring the client.
-   * - `userAgent`      : A string to append to the built in user agent string that is passed as a connection property
-   * to the service.
-   * - `retryOptions`   : The retry options for all the operations on the client/producer/consumer.
-   * A simple usage can be `{ "maxRetries": 4 }`.
-   * - `webSocketOptions`: Options for the websocket implementation used for AMQP.
+   * - `retryOptions`   : A set of options to configure the retry policy for all the operations on the client.
```

Comment:
Replace the pattern "A set of options to configure" with "Configures" IMO.

Reply:
Fixed in https://github.com/Azure/azure-sdk-for-js/pull/6736/commits/03027f8c2c6138d2b112a489ef2ea785faa54c76.